### DISCUSSION TITLE
Unkknown field name 'coordinates'

### DISCUSSION THREAD

**Discussion by uRhaineWork** [mattieserver/netbox-topology-views#207] [Q&A]
Hi,
After plugin install, I noticed a problem when saving changes in Devices section. Devices created after plugin work fine, can save, no errors. I thought that maybe moving around davice in Topo View and saving data migt work, but sadly no.
netbox 3.2.4
![obraz](https://user-images.githubusercontent.com/103448821/174599043-60ec56ee-0a48-442e-8f16-ddd6a8592fc3.png)

**Comment by swgATpj** [comment]
Look there: [https://github.com/mattieserver/netbox-topology-views/issues/121](url)
You can enable the debug option, so you can see the specific error and the location where this error happens.

**Comment by joaolucasmacedo** [comment]
I saw many people with the same issue.
